app/lib/utils/btc.js

```diff
@@ -31,20 +38,26 @@ export function bitsToBtc(bits, price) {
   if (bits === undefined || bits === null || bits === '') return null
   const sats = bits * 100
 
-  return satoshisToBtc(sats, price)
+  return satoshisToBtc(sats, price) || 0
 }
 
 export function bitsToSatoshis(bits, price) {
   if (bits === undefined || bits === null || bits === '') return null
 
-  return bits * 100
+  return bits * 100 || 0
+}
+
+export function bitsToMillisatoshis(bits, price) {
+  if (bits === undefined || bits === null || bits === '') return null
+
+  return bits * 100 * 1000 || 0
 }
 
 export function bitsToFiat(bits, price) {
   if (bits === undefined || bits === null || bits === '') return null
```

Comment:
This condition is used a lot in this module (>3 times). I would extract it into a separate function. Like isEmptyAmount. For readability and ease of refactoring later.

Reply:
This was bothering me too but thought I'd leave it out of this PR. In retrospect, give that this PR makes quite a few changes across this file it makes sense to clean that up here and now. Have added an additional refactor commit that extracts this.

Reply:
ok